src/state/sos/sos.actions.js

```diff
@@ -1,12 +1,12 @@
 /* eslint no-underscore-dangle: ['error', { 'allow': ['_id'] }] */
-import appApiClient from 'api/';
+import { appApiClient } from 'api/';
 import { getUserSecureStorage } from 'helpers';
 import * as types from './sos.types';
 
 const getContacts = (dispatch) => async () => {
   try {
-    const { username, token } = await getUserSecureStorage();
-    const response = await appApiClient.getSosContacts(username, token);
+    const { username } = await getUserSecureStorage();
```

Comment:
is this affecting also authentication actions?

Reply:
i changed my actions because sos apis changed from requiring token in params to now thats no longer needed because interceptor does that part

Reply:
and in authentication apis i dont see any case that u send the token to the server so i guess no?